.github/workflows/update-changelog.yml

```
@@ -1,51 +1,55 @@
         with:
           fetch-depth: 0
-      - name: Get version from tag
-        id: get-version
-        uses: battila7/get-version-action@v2
+          ref: ${{ github.head_ref }}
+
+      - name: Get version info
+        id: version_info
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:44: Double quote to prevent globbing and word splitting

(shellcheck)